metrics/collectors/economic/economic.go

```diff
@@ -61,8 +60,13 @@ type Collector struct {
 
 var _ prometheus.Collector = (*Collector)(nil)
 
+//go:generate mockgen -destination=./mocks/elReader.go -package=mocks github.com/Layr-Labs/eigensdk-go/metrics/collectors/economic ELReader
```

Comment:
why are you generating this? Is this mock used in the tests somewhere? Why not create a fake instead if so?

Reply:
yes check `economic_test`. The problem with fake again is, I cannot return what I want. I can only return single return value. what if my two tests expect different return value? mock let me return a value I want.

Reply:
8bbba4610bbaf3ddabad884739b743aa842ef4ec